autoscaler/cluster.py

```diff
@@ -536,7 +536,11 @@ def fulfill_pending(self, asgs, selectors_hash, pods):
 
             new_capacity = group.actual_capacity + units_requested
             if not self.dry_run:
-                async_operation = group.scale(new_capacity)
+                try:
+                    async_operation = group.scale(new_capacity)
+                except CloudError as e:
+                    logger.warn("Error while scaling Scale Set: {}".format(e.message))
```

Comment:
That seems like a good idea. How does our Sentry integration work? I see that Raven is installed as a dependency in the autoscaler, but I don't see it used anywhere